pyartcd/pyartcd/redis.py

```diff
@@ -85,3 +86,51 @@ async def delete_key(conn: aioredis.commands.Redis, key: str) -> int:
     res = await conn.delete(key)
     logger.debug('Key %s %s', key, 'deleted' if res else 'not found')
     return res
+
+
+@handle_connection
+async def list_push(conn: aioredis.commands.Redis, key: str, value: str):
+    """
+    Push to a list in Redis, like a queue (FIFO). Docs: https://redis.io/docs/data-types/lists/
+    """
+
+    logger.debug('Pushing value %s to list %s', value, key)
+    await conn.lpush(key, value)
+
+
+@handle_connection
+async def list_pop(conn: aioredis.commands.Redis, key: str) -> (str, None):
```

Comment:
This means a return type of `tuple(string, None)`. We do need to use `typing.Optional`, there's places where it is used you can take inspiration from

Reply:
like this right https://github.com/openshift-eng/art-tools/pull/90/commits/5548fd127628fab3ad7bb6b4685091107ebdb2dd